content/en/security/cspm/getting_started.md

```diff
@@ -195,6 +185,18 @@ To filter the default detection rules by cloud provider:
 
 After you explore the default detection rules, you can review and take action on your cloud misconfigurations in the [Security Findings Explorer][3], [customize how each rule scans your environment][4], and [set up notification targets][5].
 
+## Disabling CSPM
+
+When you disable CSPM, your previous findings and homepage are still available in-app, but you will not incur additional billing costs. 
```

Comment:
Trying to avoid using future tense here, feel free to disregard if it changes up the meaning to much

```suggestion
Once you've disabled CSPM, your previous findings and the homepage are still available in-app, and you do not incur additional billing costs. 
```